Babylon/groups/azure/groups/acr/__init__.py

```diff
@@ -0,0 +1,17 @@
+from click import group
+from click import pass_context
+from click.core import Context
+from .commands import list_commands
+from .groups import list_groups
+
+
+@group()
+@pass_context
+def acr(ctx: Context):
+    """Nothing needed here"""
+
```

Comment:
To use the `make_pass_decorator` in descending commands we need to add the object we want to get later in a previous context.
So the following content should be in this call instead of future commands
```python
    os.system(f"az acr login --name {acr_dest_registry_name}")
    ContainerRegistryClient(
        f"https://{acr_dest_registry_name}",
        ctx.parent.obj,
        audience="https://management.azure.com")
```

Reply:
I wish I could :'(
I had to create a new registry client because we need to log to a different registry for push and pull